scripts/code_checks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Please keep using the Makefile checks for now. This script would get blown away soon.